generated/nitclk/nitclk.proto

```diff
@@ -9,24 +9,17 @@ syntax = "proto3";
 option java_multiple_files = true;
 option java_package = "com.ni.grpc.tclk";
 option java_outer_classname = "NiTClk";
-option csharp_namespace = "NationalInstruments.Grpc.TClk";
+option csharp_namespace = "NationalInstruments.Grpc.Tclk";
 
-package nitclk_grpc;
```

Comment:
I'm not sure why this changed. This should just be based on config.py's `namespace_component` which hasn't changed. Maybe after fixing the chsarp_namespace and rebuilding it'll clear this up.